client/src/components/pages/home/HomeBanner.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+import ContentManager from "../../../ContentManager";
+import Content from "../../../content/home-en";
+import { get } from "../../../utilities";
+
+import stripesBottomLeft from "./svg/stripes-bottom-left.svg";
+import stripesTopRight from "./svg/stripes-top-right.svg";
+import stripedCircleWhite from "./svg/striped-circle-white.svg";
+import stripedCircleOrange from "./svg/striped-circle-orange.svg";
+
+import "./HomeBanner.css";
+import { Button, Typography } from "antd";
+import { useMatchMedia } from "../../../utilities";
+
+const UI = ContentManager.getUI();
+
+/**
+ * @typedef {Object} LoginProps
+ *
+ * @property {Object} user
+ * @property {Function} setUser
+ */
+/**
+ * Opens a popup window to osu! OAuth login.
+ * This is copied from LoginButton.js
+ *
+ * @param {LoginProps} props
+ */
+async function login({ user, setUser }) {
```

Comment:
use the shared util `showAuthPopup` (from utilities.js) instead of copy-pasting this

Reply:
Login button removed in 400d747fcda341fb788f49c155bcf464530e1af2